hi-ml/src/health_ml/utils/wsi_utils.py

```diff
@@ -0,0 +1,19 @@
+from async_timeout import Any
+import torch
+import numpy as np
+from torch.utils.data.dataloader import default_collate
+from typing import List
+
+
+def list_data_collate(batch: List) -> Any:
+    """
```

Comment:
The name of the function is too generic. it is only doing something for images, so could be called image_collate?